### PR TITLE
Tools - Add HEMTT SQF support and fix commas are not semicolons

### PR DESCRIPTION
**Pull request by BrettMayson**
**When merged this pull request will:**
- Running HEMTT's upcoming SQF checking / compiling on ACE, and it found some typos
- Switches `.sqf` to `.inc.sqf` for files that only intend to be included, and are invalid on their own

### IMPORTANT

- If the contribution affects [the documentation](https://github.com/acemod/ACE3/tree/master/docs), please include your changes in this pull request so the documentation will appear on the [website](https://ace3.acemod.org/).
- [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

File: addons/common/functions/fnc_checkFiles.sqf [hunk 1]
```diff
@@ -63,7 +63,7 @@ private _oldCompats = [];
 } forEach _addons;
 
 if (_oldAddons isNotEqualTo []) then {
-    _oldAddons = _oldAddons apply {"%1.pbo", _x};
+    _oldAddons = _oldAddons apply { format["%1.pbo", _x] };
```
Comment (BaerMitUmlaut) [inline addons/common/functions/fnc_checkFiles.sqf:66]: ?

Comment (BrettMayson) [reply addons/common/functions/fnc_checkFiles.sqf:66]: ?

Comment (johnb432) [inline addons/common/functions/fnc_checkFiles.sqf:66]:
```suggestion
    _oldAddons = _oldAddons apply { format ["%1.pbo", _x] };
```

Comment (BrettMayson) [reply addons/common/functions/fnc_checkFiles.sqf]: I'd actually like to run a find and replace against stuff like this, may be a minor pain on some PRs but imo worth it

Comment (BaerMitUmlaut) [reply addons/common/functions/fnc_checkFiles.sqf]: This is weird. Seems like this was the intention, but have you tested it?

Comment (BrettMayson) [reply addons/common/functions/fnc_checkFiles.sqf]: I have not tested, can't atm. But I can't see what else could've possibly been the intention. @PabstMirror Can you confirm?

Comment (johnb432) [reply addons/common/functions/fnc_checkFiles.sqf]: I'm not Pabst, but if you look at the old code https://github.com/acemod/ACE3/commit/9d1695ffe2e1f3e0665c2f0891dfa4631f862b2b#diff-40dceb2af61593c5d04c1135da9450c7853e0fb6f8513adf7dccd9baa1102497, it only listed one outdated PBO.
I imagine it was not intentional to omit the `format`.

File: addons/refuel/functions/fnc_makeJerryCan.sqf [hunk 2]
```diff
@@ -88,7 +88,7 @@ _action = [QGVAR(CheckFuel),
     {},
     [],
     [0,0,0],
-    REFUEL_ACTION_DISTANCE] call EFUNC(interact_menu,createAction),
+    REFUEL_ACTION_DISTANCE] call EFUNC(interact_menu,createAction);
```
Comment (Dystopian) [inline addons/refuel/functions/fnc_makeJerryCan.sqf:91]:
```suggestion
    REFUEL_ACTION_DISTANCE
] call EFUNC(interact_menu,createAction);
```
I think this is needed in all actions above in this file.

File: addons/common/functions/fnc_checkFiles.sqf (hunk 1 above)
Comment (PabstMirror) [reply addons/common/functions/fnc_checkFiles.sqf]: I don't think I actually wrote that but fix looks good

**Review by PabstMirror — APPROVED** [review]

**Review by jonpas — COMMENTED** [review]
Docs need to be updated for minimal HEMTT version if we're removing ASC config.

**Comment by BrettMayson** [comment]
Actually yeah I suppose it could stay for now, doesn't hurt anything

**Review by jonpas — APPROVED** [review]

**Review by jonpas — APPROVED** [review]

**Pull request merged**